portfolio/src/main/webapp/scripts/server-dev.js

```diff
@@ -12,17 +12,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+var i = 0;
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
+function getAnswer() {
+  fetch('/data')
+  .then(response => response.json())
+  .then(answ_list => {
+    document.getElementById('answer-container').innerText = 
+                                          answ_list[(i++) % 3];
```

Comment:
replace 3 with answ_list.size? or length. (sorry I forgot which is the right one)

Reply:
Done.